pipelines.yml

```diff
@@ -53,46 +53,56 @@ pipelines:
         type: Bash
         configuration:
           integrations:
-            - name: releases_jfrog
+            - name: releases_jfrog_io_deployer
+            - name: il_automation
+            - name: entplus_deployer
           inputResources:
             - name: buildJfrogCliPluginGit
           runtime:
             type: image
             image:
-              auto:
-                language: go
-                versions:
-                  - "1.15.2"
+              custom:
+                name: releases-docker.jfrog.io/jfrog-ecosystem-integration-env
+                tag: 1.4.0
           environmentVariables:
             JFROG_CLI_PLUGIN_DESCRIPTOR_FILE_NAME: "file-name-with-extension.yml"
         execution:
           onStart:
             - update_commit_status buildJfrogCliPluginGit --context "$step_name"
           onExecute:
             # Install yq to parse yaml.
-            - GO111MODULE=on go get github.com/mikefarah/yq/v3
+            - GO111MODULE=on go get github.com/mikefarah/yq/v4
```

Comment:
go get: installing executables with 'go get' in module mode is deprecated.
	Use 'go install pkg@version' instead.
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.